package-lock.json

```diff
@@ -0,0 +1,2624 @@
+{
```

Comment:
Cuidaddo com esses arquivos `package.json` e `lock.json` se não for importante pro projeto não sobe.

Reply:
Eles foram criados na adição do commitizen, por isso os subi, nesse caso não precisaria subir?

Reply:
não precisa subir esses arquivos eles são apenas para padronizar os teus commits.